server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -313,7 +313,7 @@ def validate_openshift_subdomain(deployment, subdomain)
       end
 
       def validate_hostname(deployment)
-        regex = /^[A-z0-9\.\_\-]{1,55}$/
+        regex = /^[A-z0-9\.\_\-]+$/
```

Comment:
@jmrodri I took out the length check on the server because it could cause a deployment to fail to save after attaching the domain name.  If you're still ok with the ACK after that change, I'll merge.  My test deployment of RHEV, OSE, CFME went through using max length names.